api/pacman-api-compliance/src/main/java/com/tmobile/pacman/api/compliance/repository/FilterRepositoryImpl.java

```diff
@@ -603,4 +670,92 @@ private List<String> getUntaggedAssets(String assetGroup) throws Exception {
         List<Map<String, Object>> untaggedAssetList= elasticSearchRepository.getDataFromES(assetGroup, null, mustFilter,null, null,Arrays.asList(RESOURCEID), null);
         return untaggedAssetList.stream().map(obj -> (String)obj.get(RESOURCEID)).collect(Collectors.toList());
     }
+
+    private Map<String ,Object> getDateFilter(Map<String, List<String>> filter){
+        Map<String ,Object> dateFilterMap=new HashMap<>();
+
+        Date startDate=null;
+        Date endDate=null;
+
+        List<String> dateFilter=filter.get("_loaddate");
+        if(dateFilter!=null && !dateFilter.isEmpty()){
+            List<String> dateRangeList = filter.get("_loaddate");
+            StringBuilder dateRange= new StringBuilder();
+            for(String range:dateRangeList){
+                dateRange.append(range);
+            }
+            String[] dates = dateRange.toString().split(" - ");
+            SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+            try {
+                startDate = dateFormat.parse(dates[0]);
+                endDate = dateFormat.parse(dates[1]);
+
+            } catch (ParseException e) {
+                logger.error("Error in Date Format");
+            }
+        }
+
+        String gte = null;
+        String lte = null;

```

Comment:
what is these dates?

Reply:
"Parameters for range in an Elasticsearch query:
'gte' stands for 'Greater than or equal to.'
'lte' stands for 'Less than or equal to.'"